src/Computation/IrreducibleSymbolicExpressionException.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Symbolica.Computation
+{
+    [Serializable]
+    public class IrreducibleSymbolicExpressionException : Exception
```

Comment:
Do you think we should include a field that tells us the type of thing that couldn't be reduced? Could be a string here or an Enum maybe if it's a finite set?